ntfy/Persistence/SubscriptionManager.swift

```diff
@@ -36,6 +36,30 @@ struct SubscriptionManager {
         poll(subscription) { _ in }
     }
     
+    func backgroundPoll(_ subscription: Subscription, completionHandler: @escaping ([Message]) -> Void) {
+        let user = store.getUser(baseUrl: subscription.baseUrl!)?.toBasicUser()
+        Log.d(tag, "Polling from \(subscription.urlString()) with user \(user?.username ?? "anonymous")")
+        let backgroundConfig = URLSessionConfiguration.background(withIdentifier: "com.example.myapp.background")
```

Comment:
Is this identifier correct? (Or important?)

Reply:
It can be any String, no need to be the same with app identifier